content/IA-update_connections/Overview.textile

```diff
@@ -0,0 +1,293 @@
+---
+title: Connections
+meta_description: "Establish and maintain a persistent connection to Ably using the Realtime Client Library SDK."
+section: realtime
+index: 20
+languages:
+  - javascript
+  - nodejs
+  - ruby
+  - java
+  - swift
+  - objc
+  - csharp
+  - flutter
+jump_to:
+  Help with:
+    - Introduction#intro
+    - Getting started#title
+    - Connection state explained
+    - Handling failures
+    - Connection state recovery
+    - Heartbeats#heartbeats
+    - API Reference#api-reference
+
+
+what are connections, mention states exist
+(repeating box) realtime only
+Create a connection
+(caveat box) connection is implicit with instantiation but this section cover explicit call to connect + usecase for when to use this option
+connect method + code example
+Monitor connections to an app
+From FAQ
+Connection metachannel and limitations of it. This will also cover dev console.
+Connection IDs
+(caveat box) Connections across multiple browser tabs
+Heartbeats
+explanation
+Code example + ping
+Close a connection
+explanation, incl 2min delay if not explicitly closed = peak rate + code example
+---
+
+h2(#intro). Introduction
```

Comment:
I don't think this section heading is required, and it's also not in Miro.

Reply:
Done